src/main/java/co/rsk/bitcoinj/core/Utils.java

```diff
@@ -77,12 +86,17 @@ public static byte[] bigIntegerToBytes(BigInteger b, int numBytes) {
         if (b == null) {
             return null;
         }
```

Comment:
this is not part of bitcoinj's original code, but part of ours.
I think is better to keep it there, and add the bitcoinj's checks after.